components/styles.tsx

```diff
@@ -0,0 +1,23 @@
+import { StyleSheet } from "react-native"
```

Comment:
Love styling <3 , just wait until our manager will approve it

Reply:
Looks really solid keep up the good work!